src/main/java/org/jenkinsci/plugins/docker/commons/tools/DockerToolInstaller.java

```diff
@@ -0,0 +1,118 @@
+package org.jenkinsci.plugins.docker.commons.tools;
```

Comment:
Marked as a :bug: for a license header, are you kidding ?
project has a LICENSE.txt file on scm root, why do we need such a license header ?
Do you add license headers to icon files as well ? :-P